Rep 2102 adfs sp

kietlieng opened the pull request.

kietlieng committed:
Adding content parameter for on the fly adding of metadata file.

Ticket: REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng committed:
Fixing my mistakes on class

Ticket: REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng committed:
Fixing braces

Ticket: REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng committed:
Getting rid of php-cs-fixer.phar since it's causing the test to fail
on on library code format.

Ticket: REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng committed:
Taking out old php versions that we don't need to test against

Ticket: REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng committed:
Creating new EntityDescriptorStore

Ticket:REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng committed:
Adding unit testing fro XmlEntityDescripter

Ticket: REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:

kietlieng commented:
@johnyhuh  please PR.  

I added some functional test and separated the parameter into it's own file.  I still think it doesn't make sense but I if there's a chance that it get's merged into the public repo I'll aim for that.  Keeps us from diverging too much.

johnyhuh commented on src/LightSaml/Provider/EntityDescriptor/XmlEntityDescriptorProvider.php:
```diff
@@ -0,0 +1,47 @@
+    private $entityDescriptor;
+
+    /**
+     * @param string $filename
```
Make docblock consistent with the argument.

johnyhuh commented on src/LightSaml/Store/EntityDescriptor/XmlEntityDescriptorStore.php:
```diff
@@ -0,0 +1,93 @@
+    private $object;
+
+    /**
+     * @param string $filename
```
Make docblock consistent with the argument.

kietlieng committed:
Update docblockk

Ticket:REP-2102_adfs_sp
Purpose of Change:
Description of Change:
Tags:

Conflicts Resolved:
